doc/usage.rst

```diff
@@ -422,7 +422,7 @@ linked via the ``:intermediate:`` RQT-items:
     the righthand side are used to link the *intermediate* items to the *target* items.
     External relationships are not compatible with this feature (yet).
 
-:intermediate: *optional*, *single argument*
+:intermediate: *required*, *single argument*
```

Comment:
This option is obviously not required when you don't want to link via intermediate items. Is this change fine or do you know a better way to document this requirement?

Reply:
So you want to underline that single argument is required, not the option?

```suggestion
:intermediate: *optional*, *single argument* (*required* when used)
```

Reply:
Or maybe 
```suggestion
:intermediate: *optional* (*required* when type includes ``|``), *single argument*
```